com.vrcfury.vrcfury/Editor/VF/Feature/OnDemandSyncBuilder.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using UnityEditor;
+using UnityEngine.UIElements;
+using VF.Feature.Base;
+using VF.Injector;
+using VF.Inspector;
+using VF.Model.Feature;
+using VF.Service;
+using VF.Utils;
+using VF.Utils.Controller;
+using VRC.SDK3.Avatars.ScriptableObjects;
+
+namespace Editor.VF.Feature {
+    public class OnDemandSyncBuilder : FeatureBuilder<OnDemandSync> {
+        [VFAutowired] private readonly OnDemandSyncService onDemandSyncService;
+
+        [FeatureBuilderAction]
```

Comment:
This may need a FeatureOrder, since as currently written, this is applied when the component is processed at the "Default" order, which could be before toggles or other things are added.

Reply:
Added a new FeatureOrder right after Default, so that it runs after Toggles and Full Controllers are merged